src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -59,6 +60,22 @@ internal static class ProtocolConversions
             { WellKnownTags.NuGet, LSP.CompletionItemKind.Text }
         };
 
+        public static Uri GetUriFromFilePath(string filePath)
+        {
+            if (filePath is null)
+            {
+                throw new ArgumentNullException(nameof(filePath));
+            }
+
+            // Remove preceding slash if we're on Window as it's an invalid URI.
```

Comment:
Let's point this to https://github.com/dotnet/aspnetcore/issues/19948 now that the bug has been filed.